documentation/developer_updates.md

```diff
@@ -5,20 +5,171 @@ we're making.
 
 ## November 2020
 
-### Puppet 7 Release Prep
+### Changes coming in Bolt 3.0
+
+With the new year fast approaching, the team is preparing to release the next
+major version of Bolt: **Bolt 3.0**! Over the next few releases, you will start
+to see deprecation warnings for specific features and configuration options
+that are slated for removal in Bolt 3.0. To help you prepare for the release,
+we've compiled a list of expected changes and removals.
+
+#### Changes
+
+- **Ship with Puppet 7**
+
+  Bolt packages will ship with Puppet 7. This change should not impact most
+  users, but changes to the plan language in Puppet may require updating your
+  plans.
+
+  To read more about the changes in Puppet 7, see the [Puppet 7 release
+  notes](https://puppet.com/docs/puppet/7.0/release_notes_puppet.html#release_notes_puppet_x-0-0).
+
+- **Ship with Ruby 2.7**
+
+  Bolt packages will ship with Ruby 2.7 instead of Ruby 2.5.1. This change will
+  not affect most users who install the Bolt package, as Bolt packages ship with
+  their own Ruby. However, gem installs will require Ruby 2.7 or higher, and
+  users who install gems with Bolt's Ruby may also need to update their gems.
+
+- **New default modulepath**
+
+  Since the introduction of the module feature in Bolt 2.30.0, Bolt has used a
+  shorter, simpler default modulepath when a project enables the feature.
+  Starting in Bolt 3.0, the module management feature will be enabled for all
+  projects, meaning the default modulepath will also be updated.
```

Comment:
It seems useful to add that users can update using `bolt project migrate`, either now or when 3.0 is released.

Reply:
Heh, I actually had that in here originally but pulled it out. :P I feel like this happens all the time.